iree/compiler/Codegen/Common/test/tile_and_distribute_to_workgroups.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: iree-opt -pass-pipeline='hal.executable(hal.executable.variant(builtin.module(func.func(iree-codegen-tile-and-distribute-to-workgroups)))), canonicalize, cse' -split-input-file %s | FileCheck %s
```

Comment:
Would you mind just splitting this test into two parts then. Basically the result of `iree-codegen-insert-distribution-info` should become the input MLIR for `iree-codegen-tile-and-distribute-to-workgroups`.

Reply:
I think we don't have to change the input MLIR for `iree-codegen-tile-and-distribute-to-workgroups`. We can test the result IR and remove the check about `hal.executable.entry_point`.